richtext-ui/src/commonMain/kotlin/com/halilibo/richtext/ui/string/Text.kt

```diff
@@ -234,6 +232,14 @@ private fun AnnotatedString.animateAlphas(
   }
   var remainingText = this
   val modifiedTextSnippets = mutableStateListOf<AnnotatedString>()
+  val inlineContentSpans = getStringAnnotations(0, length)
+    .filter { it.tag == "androidx.compose.foundation.text.inlineContent" }
+  val inlineContentStyles = spanStyles.filter { it.tag == "androidx.compose.foundation.text.inlineContent"}
+  val inlineSpansToWatchOutFor = inlineContentSpans + inlineContentStyles
+  // Chopping up a string with inline content in it causes a crash. So we need to fade in the entire block.
+  if (inlineSpansToWatchOutFor.isNotEmpty()) {
```

Comment:
To me, this reads a bit easier because it's clear that you're doing something if either are not empty. `inlineSpansToWatchOutFor` threw me off because you're creating a List<Any> by appending two separate things together than then simply checking if it's empty. Wdyt?

```suggestion
  // Chopping up a string with inline content in it causes a crash. So we need to fade in the entire block.
  if (inlineContentSpans.isNotEmpty() || inlineContentStyles.isNotEmpty()) {
```

Reply:
definitely. I had the list used later in an earlier test but not useful anymore